datajoint/fetch.py

```diff
@@ -37,9 +38,15 @@ def _initialize_behavior(self):
     @property
     def squeeze(self):
         """
+        DEPRECATED
```

Comment:
Perhaps we could remove `FetchBase` and `Fetch1` and make `Fetch` do everything.  `Fetch` would then have the argument `exactly_one`.  The `fetch1` method could simply call `fetch(*args, exactly_one=True, **kwargs)`.  This could obviate the inheritance and the state handling.